backend/pkg/modules/internal/spec_differ/spec_diffs.go

```diff
@@ -180,10 +179,14 @@ func (s *specDiffer) addDiffToSend(diff *_spec.APIDiff, modifiedPathItem, origin
 	}
 
 	var specTimestamp time.Time
-	if specType == common.PROVIDED {
+	switch specType {
+	case common.PROVIDED:
 		specTimestamp = time.Time(apiInfo.ProvidedSpecCreatedAt)
-	} else {
+	case common.RECONSTRUCTED:
 		specTimestamp = time.Time(apiInfo.ReconstructedSpecCreatedAt)
+	default:
+		log.Warnf("Unknown spec type: %v", specType)
```

Comment:
please state in the warning that we are using the `ProvidedSpecCreatedAt` - BTW are we share that it is exists? can we use it if we don't know that spec type?

Reply:
It is not critical we will always have it probably